Pluggy.SDK/Model/Webhook.cs

```diff
@@ -51,9 +51,13 @@ public WebhookEvent Event
 
     public class WebhookEventPayload
     {
+        [Obsolete("Use EventId instead", true)]
```

Comment:
```suggestion
        [Obsolete("Use EventId instead", false)]
```
Setting it to `true` will cause compilation errors.